blazar/db/migration/alembic_migrations/versions/ee3b2513b59f_rename_extra_capability.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2024 OpenStack Foundation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""rename extra capability to resource property
+
+Revision ID: ee3b2513b59f
+Revises: d5a379ff6ba3
+Create Date: 2024-01-06 00:22:00.128435
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'ee3b2513b59f'
+down_revision = 'd5a379ff6ba3'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    # Drop existing foreign key constraints
+    op.drop_constraint('computehost_extra_capabilities_ibfk_1',
+                       'computehost_extra_capabilities',
+                       type_='foreignkey')
+    op.drop_constraint('networksegment_extra_capabilities_ibfk_1',
+                       'networksegment_extra_capabilities',
+                       type_='foreignkey')
+    op.drop_constraint('device_extra_capabilities_ibfk_1',
+                       'device_extra_capabilities',
+                       type_='foreignkey')
+    # Rename the ExtraCapability model to ResourceProperty
+    op.rename_table('extra_capabilities', 'resource_properties')
+    # Update the column names and types
+    op.alter_column('resource_properties', 'capability_name',
```

Comment:
Should these tables also not be renamed?

Reply:
Yes the table name is `resource_properties ` - https://github.com/openstack/blazar/blob/stable/yoga/blazar/db/migration/alembic_migrations/versions/02e2f2186d98_resource_property.py#L36C5-L36C43
The column `capability_name` is [renamed](https://github.com/openstack/blazar/blob/stable/yoga/blazar/db/migration/alembic_migrations/versions/02e2f2186d98_resource_property.py#L39) to `property_name` in the table `resource_properties` as the earlier operation renames the table.